source/main/resources/CacheSystem.cpp

```diff
@@ -1106,7 +1106,9 @@ bool CacheSystem::CheckResourceLoaded(Ogre::String & filename, Ogre::String& gro
     if (ResourceGroupManager::getSingleton().resourceExistsInAnyGroup(filename))
     {
         group = ResourceGroupManager::getSingleton().findGroupContainingResource(filename);
-        return true;
+        // Files are not allowed to be loaded from the Content resource group, except odef files
+        if(group != "Content" || filename.find("odef") != std::string::npos)

```

Comment:
Umm, neat hack.

I can't recall why we have both CONTENT_RG "Content" + TEMP_RG "Temp" while both are only used during cache regen. I think CONTENT_RG should be destroyed afterwards, in which case we wouldn't need this check.